closed/src/java.base/share/classes/com/sun/crypto/provider/NativeChaCha20Cipher.java

```diff
@@ -1001,15 +1035,16 @@ public synchronized int doUpdate(byte[] in, int inOff, int inLen, byte[] out,
         @Override
         public synchronized int doFinal(byte[] in, int inOff, int inLen, byte[] out,
                 int outOff) throws ShortBufferException, KeyException {
+            if ((in == null) && (inLen != 0)) {
+                throw new ShortBufferException("Input buffer in null, while input length is non-zero.");
```

Comment:
I think this should say "is" instead of "in".

Reply:
You're right. Changed.